openmc/universe.py

```diff
@@ -402,22 +402,23 @@ def create_xml_subelement(self, xml_element):
         if len(self._name) > 0:
             element.set("name", str(self._name))
 
-        if isinstance(self._fill, openmc.Material):
-            element.set("material", str(self._fill._id))
+        if isinstance(self.fill, basestring):
+            assert self.fill.strip().lower() == 'void'
```

Comment:
The value is checked in the setter.  I just thought I'd add this extra line of defense in case one of us accidentally bypasses the setter.  If you don't like it I'll remove it without complaint